microsite/pages/en/plugins.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const fs = require('fs');
+const yaml = require('js-yaml');
+const React = require('react');
+const Components = require(`${process.cwd()}/core/Components.js`);
+const {
+  Block: { Container },
+  BulletLine,
+} = Components;
+
+const pluginsDirectory = require('path').join(process.cwd(), 'data/plugins');
+const pluginMetadata = fs
+  .readdirSync(pluginsDirectory)
+  .map(file =>
+    yaml.safeLoad(fs.readFileSync(`./data/plugins/${file}`, 'utf8')),
+  );
+const truncate = text =>
+  text.length > 170 ? text.substr(0, 170) + '...' : text;
+
+const addPluginDocsLink =
+  'http://localhost:3000/docs/plugins/add-to-marketplace';
+const defaultIconUrl = 'img/logo-gradient-on-dark.svg';
+
+const Plugins = () => (
+  <main className="MainContent">
+    <div className="PluginPageLayout">
+      <div className="PluginPageHeader">
+        <h2>Plugins</h2>
+        <span>
+          <a
+            className="PluginAddNewButton ButtonFilled"
+            href={addPluginDocsLink}
+          >
+            <b>Add Plugin</b>
+          </a>
+        </span>
+      </div>
+      <BulletLine style={{ width: '100% ' }} />
+      <Container wrapped className="grid">
+        {pluginMetadata.map(
+          ({
+            iconUrl,
+            title,
+            description,
+            author,
+            documentation,
+            category,
+          }) => (
+            <div className="PluginCard">
+              <div className="PluginCardHeader">
+                <img src={iconUrl || defaultIconUrl} alt={title} />
+                <h2 className="PluginCardTitle">{title}</h2>
+                <p style={{ marginBottom: '0.25em' }}>by {author}</p>
+                <span className="ChipOutlined">{category}</span>
```

Comment:
It was mentioned to me that we might allow a tags list for plugins and that category might be rendered along with these (visually as the first tag but as a separate field). I've placed the category here because it works best with the current data but if we were to add tags I would expect it to move into the body (probabably at the end like the explore page in backstage).